benchmark/scenarios/test/test_write_request_load.py

```diff
@@ -0,0 +1,362 @@
+from twisted.trial.unittest import SynchronousTestCase
+
+from itertools import repeat
+from uuid import uuid4
+from ipaddr import IPAddress
+from twisted.internet.defer import succeed, Deferred
+from twisted.internet.task import Clock
+from twisted.python.components import proxyForInterface
+from twisted.trial.unittest import SynchronousTestCase
+
+from flocker.apiclient._client import (
+    IFlockerAPIV1Client, FakeFlockerClient, Node
+)
+
+from benchmark.scenarios import (
+    WriteRequestLoadScenario, WRateMeasurer, WRequestRateTooLow,
+    WRequestRateNotReached, WRequestOverload, WDataseCreationTimeout
+)
+
+from benchmark.scenarios.read_request_load import DEFAULT_SAMPLE_SIZE
+
+from benchmark.cluster import BenchmarkCluster
```

Comment:
I would group all benchmark imports together, i.e. I'd remove blank lines between them.
Also, I prefer to alphabetically sort imports within a group, so I would put the import from `benchmark.cluster` before `benchmark.scenarios`.

Also, maybe these imports should be relative rather than absolute?